Packs/ThreatConnect/ReleaseNotes/2_0_6.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### ThreatConnect v2
+- Fixed an issue where fetching indicators from multiple owners did not work as intended.
```

Comment:
`fetching indicators from multiple owners is now supported.`

@kirbles19 what do you think?

Reply:
Added support for fetching indicators from multiple owners.